mirrord/config/src/config/depricated.rs

```diff
@@ -0,0 +1,34 @@
+use crate::config::source::MirrordConfigSource;
+
+#[derive(Clone)]
+pub struct Depricated<T>(String, T);
+
+impl<T> Depricated<T> {
+    pub fn new(message: &'static str, inner: T) -> Self {
+        Depricated(message.to_owned(), inner)
+    }
+
+    pub fn untagged(container: &'static str, attr: &'static str, inner: T) -> Self {
```

Comment:
I don't think this should be usable without specifying a. when it will be removed and b. what should be used instead